src/openslide-vendor-dicom.c

```diff
@@ -718,9 +689,33 @@ static bool maybe_add_file(openslide_t *osr,
     return true;
   }
 
-  // check the image format
-  if (!get_format(f->metadata, &f->format, err)) {
-    return false;
+  // check the other image format tags, ignore DICOMs with formats we don't
+  // support
+  switch (f->format) {
+  case FORMAT_JPEG2000:
+    if (verify_tag_str(f->metadata, PhotometricInterpretation, "YBR_ICT")) {
+      f->jp2k_colourspace = OPENSLIDE_JP2K_YCBCR;
+    } else if (verify_tag_str(f->metadata, PhotometricInterpretation, "RGB")) {
+      f->jp2k_colourspace = OPENSLIDE_JP2K_RGB;
+    } else {
+      return true;
```

Comment:
These new `return true` statements ignore files with problems, where the old code would set an error and fail.  Let's be sure to do the latter (and update the comment).

Reply:
OK, done.